frontend/src/components/UnlabeledDataStatus/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class UnlabeledDataStatus extends React.Component {
+    componentDidMount() {
+        this.props.getUnlabeled();
+    }
+
+    render() {
+        const { unlabeled_data } = this.props;
+
+        console.log(this.props);
+        return (
+            <p className="unlabeled-status">
+                Unlabeled Outstanding: {unlabeled_data.length}
```

Comment:
![Screen Shot 2021-08-18 at 1 49 38 PM](https://user-images.githubusercontent.com/10951569/129948669-bd92de40-8f04-47b6-92a9-d753fdf3eb4a.png)

So I ran into this situation where I made a ton of labels and accidentally forgot to set the batch size to something reasonable. You'll notice that the unlabeled count is 0, probably because everything is assigned. I think we might need to split this count into a few parts.

1. Unlabeled and Unassigned
2. Partially labeled IRR
3. Waiting for Admin Review (IRR or Skipped)
4. With final label

Might want to move this to the details page though. Thoughts?

Reply:
Ok that sounds like a good next step! I can get working on adding all of those